waspc/src/Wasp/Psl/Valid.hs

```diff
@@ -0,0 +1,89 @@
+module Wasp.Psl.Valid where
+
+import qualified Wasp.AppSpec.App.Db as AS
+import Wasp.Project.Db
+  ( DbSystemParseError (..),
+    getDbSystemFromPrismaSchema,
+    isDbUrlInPrismaSchemaValid,
+    validDbUrlInPrismaSchema,
+  )
+import qualified Wasp.Psl.Ast.Argument as Psl.Argument
+import qualified Wasp.Psl.Ast.ConfigBlock as Psl.ConfigBlock
+import Wasp.Psl.Ast.Schema as Psl.Schema
+import Wasp.Psl.Util (findPrismaConfigBlockKeyValuePair)
+import Wasp.Valid (ValidationError (..))
+import qualified Wasp.Valid as Valid
+
+validatePrismaSchema :: Psl.Schema.Schema -> [Valid.ValidationError]
```

Comment:
I would consider puting one top level comment explaining the prupose of this module/file. I think we have one in AppSpec/Valid.hs? If so, you can look at that one for inspiration. If not, you can write a new one. You now know why we have this Valid.hs and what are teh assumptions/expectations around it, but new people be able to figure it out so easily.

Reply:
I've written a comment why do we need to validate the Prisma schema file.

I kinda like having everything in one place, it's all for the Prisma file even though some of the stuff is Prisma specific. It makes it so that when we interact with the Prisma Schema AST when constructing the AppSpec there are some assurances and we can relax.

Reply:
I do get that, but it also means that if I want to use this PSL in another part of teh system, I might be doing validations that are potentially not neccessary. But ok, let's leave it as it is, we can adjust this if it will be needed in the future!
You can resolve this comment.